apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/runnable/ProcessTransactionsThread.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c)  2018-2020. Apollo Foundation.
+ */
+
+package com.apollocurrency.aplwallet.apl.core.app.runnable;
+
+import javax.enterprise.inject.spi.CDI;
+import java.util.Collections;
+import java.util.Objects;
+
+import com.apollocurrency.aplwallet.apl.core.app.AplException;
+import com.apollocurrency.aplwallet.apl.core.app.BlockchainProcessor;
+import com.apollocurrency.aplwallet.apl.core.app.BlockchainProcessorImpl;
+import com.apollocurrency.aplwallet.apl.core.app.TransactionProcessor;
+import com.apollocurrency.aplwallet.apl.core.chainid.BlockchainConfig;
+import com.apollocurrency.aplwallet.apl.core.dao.appdata.UnconfirmedTransactionTable;
+import com.apollocurrency.aplwallet.apl.core.peer.Peer;
+import com.apollocurrency.aplwallet.apl.core.peer.PeerState;
+import com.apollocurrency.aplwallet.apl.core.peer.PeersService;
+import com.apollocurrency.aplwallet.apl.util.JSON;
+import lombok.extern.slf4j.Slf4j;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+@Slf4j
+public class ProcessTransactionsThread implements Runnable {
```

Comment:
Its not a thread but just a job/task

Reply:
Minor issue, because they run as thread / runnable, so that doesn't matter in general.